examples/python/extract-logs-write-to-cloud-logging/main.py

```diff
@@ -0,0 +1,210 @@
+import json
+from collections import defaultdict
+
+import looker_sdk
+from looker_sdk import models40 as models
+
+from google.cloud import logging
+
+sdk = looker_sdk.init40()
+
+
+def create_query():
+    response = sdk.create_query(
+        body=models.WriteQuery(
+            model="system__activity",
+            view="event_attribute",
+            fields=[
+                "event.id",
+                "event.name",
+                "event.category",
+                "event.sudo_user_id",
+                "event.created_time",
+                "user.email",
+                "user.name",
+                "permission_set.permissions",
+                "permission_set.name",
+                "permission_set.id",
+                "model_set.models",
+                "model_set.name",
+                "event_attribute.name",
+                "event_attribute.value",
+                "event_attribute.id",
+                "group.id",
+                "group.name",
+                "group.external_group_id",
+                "model_set.id",
+                "user.dev_branch_name"
+            ],
+            filters={"event.created_time": "10 minutes"},
+            sorts=["event.created_time desc"],
+            filter_config={"event.created_time": [{
```

Comment:
This should be set to null unless i'm misunderstanding behavior or api docs. 

https://developers.looker.com/api/explorer/4.0/types/Query/WriteQuery
```The filter_config represents the state of the filter UI on the explore page for a given query. When running a query via the Looker UI, this parameter takes precedence over "filters". When creating a query or modifying an existing query, "filter_config" should be set to null. Setting it to any other value could cause unexpected filtering behavior. The format should be considered opaque.```

Reply:
true. having both `filters` & `filter_config` is redundant